crypto/keyring/signing_algorithms_test.go

```diff
@@ -4,9 +4,8 @@ import (
 	"fmt"
 	"testing"
 
-	"github.com/stretchr/testify/require"
-
 	"github.com/cosmos/cosmos-sdk/crypto/hd"
+	"github.com/stretchr/testify/require"
```

Comment:
please revert this import move. 

We follow

std libs

third party libs

internal libs


seems like it moved around in other files as well